etlman/templates/projects/list_pipeline.html

```diff
@@ -29,7 +29,11 @@ <h1>{{ current_project.name }} Pipelines</h1>
             {% endif %}
             <td>
                 <a href="#"> <i class="fas fa-edit"></i></a>
-                <a href="#"> <i class="fas fa-trash"></i></a>
+                <a
+                    href="{% url 'projects:confirm_delete_pipeline' project_id=pipeline.project.id pipeline_id=pipeline.id %}">
+                    <i class="fas fa-trash"></i></a>
+                <!-- <a href="{% url 'projects:delete_pipeline' project_id=pipeline.project.id pipeline_id=pipeline.id %}">
```

Comment:
Please remove (or restore if applicable) commented out code. We can always restore it from the git history if it's needed.

Reply:
Nice catch 😅